helm/kvm-operator-chart/templates/psp.yaml

```diff
@@ -5,16 +5,24 @@ metadata:
 spec:
   privileged: false
   fsGroup:
-    rule: RunAsAny
+    rule: MustRunAs
+    ranges:
+      - min: 1
+        max: 65535 
   runAsUser:
-    rule: RunAsAny
+    rule: MustRunAsNonRoot
+  runAsGroup:
+    rule: MustRunAs
+    ranges:
+      - min: 1
+        max: 65535
   seLinux:
     rule: RunAsAny
   supplementalGroups:
     rule: RunAsAny
   volumes:
-    - 'secret'
```

Comment:
For my interest why don't we need secret? Is there no volume?

Reply:
Yup that's correct, only the configmap is mounted as a volume: https://github.com/giantswarm/kvm-operator/blob/master/helm/kvm-operator-chart/templates/deployment.yaml#L20

Reply:
Cool thanks for the explanation and good to clean up. 👍